experiment/commenter/main.go

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Commenter provides a way to --query for issues and append a --comment to matches.
+//
+// The --user and --token determines who interacts with github.
+// By default commenter runs in dry mode, add --wet to make it leave comments.
+// The --updated, --include-closed, --ceiling options provide minor safeguards
+// around leaving excessive comments.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"k8s.io/test-infra/prow/github"
```

Comment:
Why use this instead of `go-github`? I thought the prow client was private, but I guess I never set that...